staging/src/k8s.io/noderesourcetopology-api/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
```

Comment:
is there anyone else that wants to come forward for approver in seeding this repo?

Reply:
@marquiz expressed interest in being a reviewer. I think we are good to go.